src/Microsoft.Health.Fhir.Azure/IntegrationDataStore/AzureBlobIntegrationDataStoreClient.cs

```diff
@@ -178,6 +178,13 @@ await _integrationStoreRetryExceptionPolicyFactory
                                 return result;
                             });
             }
+            catch (AggregateException ex) when (ex.InnerExceptions[0] is RequestFailedException)
+            {
+                var innerException = (RequestFailedException)ex.InnerExceptions[0];
+                _logger.LogInformation(innerException, "{Error}", innerException.Message);
```

Comment:
Looks like its already logged in:

https://github.com/microsoft/fhir-server/blob/8b9207c2712121558744ad3be5fec6af27e0eacf/src/Microsoft.Health.Fhir.Azure/IntegrationDataStore/AzureBlobIntegrationDataStoreClient.cs#L248-L261

Reply:
removed redundant logging